thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/analysis-options.ftl

```diff
@@ -0,0 +1,47 @@
+<div class="analysis-card padding-all">
+    <div class="analysis-title bottom-buffer">Trend Analysis
```

Comment:
To make this more semantic, perhaps consider adding header tag to "Trend Analysis". For e.g. <h2>Trend Analysis</h2>